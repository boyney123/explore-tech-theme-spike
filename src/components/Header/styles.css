@media screen and (max-width: 768px) {
  .section-header .title {
    font-size: 2em;
  }
  .section-header .subtitle {
    font-size: 1.5em;
  }
}

.section-header .button {
  margin-bottom: 0;
}

.section-header .button i {
  margin-right: 10px;
}
