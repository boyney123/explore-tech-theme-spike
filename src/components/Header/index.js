import React from 'react'

import './styles.css'

const Header = ({ title, subtitle, contribute }) => {
  return (
    <section className="main hero section-header">
      <div className="hero-body">
        <div className="container has-text-centered">
          <h1 className="title">{title}</h1>
          {subtitle && <h2 className="subtitle">{subtitle}</h2>}
        </div>
      </div>
    </section>
  )
}

export default Header
