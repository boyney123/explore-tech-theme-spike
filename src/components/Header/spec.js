import React from 'react'
import { shallow } from 'enzyme'

import Header from './'

describe('Header', () => {
  it('renders correctly with the given title and subtitle', () => {
    const wrapper = shallow(<Header title="Title" subtitle="Subtitle" />)
    expect(wrapper).toMatchSnapshot()
  })
})
