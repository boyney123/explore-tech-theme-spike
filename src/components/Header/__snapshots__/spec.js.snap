// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly with the given title and subtitle 1`] = `
<section
  className="main hero section-header"
>
  <div
    className="hero-body"
  >
    <div
      className="container has-text-centered"
    >
      <h1
        className="title"
      >
        Title
      </h1>
      <h2
        className="subtitle"
      >
        Subtitle
      </h2>
    </div>
  </div>
</section>
`;
